src/core/assets/illustrations/index.css

```diff
@@ -0,0 +1,4 @@
+:root {
```

Comment:
We usually have these customisations on the component folder, could you move them to `/components/illustrations`? 🙏🏻 
Also we usually add a short description to the property like:
```
/* Z-index property of the dialog overlay */
```
In that way, we might be able to display the property key, value & description with a simple parser on the ODS doc